Src/Coravel/Scheduling/Schedule/Interfaces/IScheduler.cs

```diff
@@ -39,6 +39,14 @@ public interface IScheduler
         /// <returns></returns>
         IScheduleInterval ScheduleWithParams<T>(params object[] parameters) where T : IInvocable;
 
+        /// <summary>
+        /// Schedule an Invocable job with a list of parameters for the specified <paramref name="invocableType"/>.
+        /// Parameters are injected into the constructor of the Invocable while remaining dependencies are resolved from DI.
+        /// </summary>
+        /// <param name="invocableType">Type of the invocable.</param>
+        /// <param name="parameters">Parameters to inject.</param>
+        IScheduleInterval ScheduleWithParams(Type invocableType, params object[ ] parameters);
```

Comment:
Minor code style: Could you remove the space within the `[ ]` here too please? Thanks! 🙂

Reply:
Resolved 😄.